tests/unit/test-date-helpers.c

```diff
@@ -45,27 +45,6 @@ main ()
   TEST_ASSERT (ecma_date_time_within_day (42.51) == 42.51);
   TEST_ASSERT (ecma_date_time_within_day (MS_PER_DAY + 42) == 42);
 
-  /* int ecma_date_days_in_year (year) */
-
-  TEST_ASSERT (ecma_date_days_in_year (0) == 366);
-  TEST_ASSERT (ecma_date_days_in_year (1600) == 366);
-  TEST_ASSERT (ecma_date_days_in_year (1603) == 365);
-  TEST_ASSERT (ecma_date_days_in_year (1900) == 365);
-  TEST_ASSERT (ecma_date_days_in_year (1970) == 365);
-  TEST_ASSERT (ecma_date_days_in_year (2000) == 366);
-  TEST_ASSERT (ecma_date_days_in_year (2000.45) == 366);
-  TEST_ASSERT (ecma_date_days_in_year (2012) == 366);
-  TEST_ASSERT (ecma_date_days_in_year (2015) == 365);
-  TEST_ASSERT (ecma_date_days_in_year (285616 + 1970) == 365);
-  TEST_ASSERT (ecma_date_days_in_year (-1970) == 365);
-
-  /* int ecma_date_day_from_year (year) */
-
-  TEST_ASSERT (ecma_date_day_from_year (1969) == -365);
-  TEST_ASSERT (ecma_date_day_from_year (1970) == 0);
-  TEST_ASSERT (ecma_date_day_from_year (1971) == 365);
-  TEST_ASSERT (ecma_date_day_from_year (2000) == 10957);
```

Comment:
There some fixme comments in this file for those functions you removed from the header. Please remove the comments too.